test/unit/presenters/formats/popular_links_format_presenter_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+class PopularLinksFormatPresenterTest < ActiveSupport::TestCase
```

Comment:
I think we might be missing a "be valid against schema" test (looking at the other format presenter tests).